scm/test_helper.go

```diff
@@ -2,24 +2,30 @@ package scm
 
 import (
 	"context"
+	"testing"
 
 	"github.com/beatlabs/github-auth/app"
+	"github.com/quickfeed/quickfeed/internal/env"
 )
 
-func TestSCMManager() *Manager {
+// TestSCMManager sets the current provider to "fake", creates a "test" organization
+// and a fake scm client for this organization.
+func TestSCMManager(t *testing.T) (SCM, *Manager) {
```

Comment:
I think the test helper should be called something different; the `Test` prefix should be reserved for actual tests, especially for things like this one, since it takes a `testing.T` as an argument.

Reply:
MockSCMManager?

Reply:
Yes, that's fine.